components/brave_wallet_ui/utils/balance-utils.ts

```diff
@@ -90,3 +93,7 @@ export function setBalance(
   const chainBalances = accountBalances.chains[chainId]
   chainBalances.tokenBalances[contractAddress.toLowerCase()] = balance
 }
+
+export function CreateEmptyTokenBalancesRegistry() {
+  return structuredClone(EmptyTokenBalancesRegistry)
```

Comment:
Why not simply return this:

```
return {
  accounts: {}
}
```

`EmptyTokenBalancesRegistry` has no other use AFAICT.